client/packets/PacketHandler.cpp

```
@@ -418,3 +418,38 @@ int packet::ProjectileDestroyHandler::handlePacket(client::Client &client,
 
   return packet::OK;
 }
+
+int packet::PositionPlayerHandler::handlePacket(client::Client &client,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)